pippo-core/src/main/java/ro/pippo/core/Application.java

```diff
@@ -268,6 +270,13 @@ public void addRoute(Route route) {
         getRouter().addRoute(route);
     }
 
+    public void addGroup(RouteGroup routeGroup) {
+        routeGroup.getRoutes().forEach(this::addRoute);
```

Comment:
Add a new method `addRouteGroup` in `Router` interface, call `getRouter().addRouteGroup()` here and move the implementation in `DefaultRouter`. The idea is that I want to add the tests in Router.